cypress/integration/login.js

```diff
@@ -24,4 +24,12 @@ describe('Sourcebooks login', function() {
         cy.get('.user-info__title').contains('Demo User');
         cy.get('.main-nav').find('li').should('have.length', 1);
     })
+
+    //test

```

Comment:
This comment doesn't bring much info about test, I suggest removing this comment because "Should validate todays date" is enough to know what this code does :)